### PR TITLE
Changelog Aug 22, 2024

### PR DESCRIPTION
**Pull request by ggodeke**
# Description

Changelog update
# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/changelog.md [hunk 1]
```diff
@@ -8,6 +8,37 @@ keywords: [changelog, upgrades, updates, releases]
...
+* Upload the file as a new hypertable within your service
+Look for the `Import data from .csv` tile in the `Import your data` step of service creation.
+
+### CloudWatch Permissions via Role Assumption
+We've released permission granting via role assumption to CloudWatch. Role assumption is both more secure and more convenient for customers who no longer need to rotate credentials and update their exporter config.
+
+For more details take a look at [our documentation](https://docs.timescale.com/use-timescale/latest/metrics-logging/integrations/).
+
+### Replica lag
+Customers now have more visibility into the state of replicas running on Timescale Cloud. We’ve released a new parameter called Replica Lag within the Operations tab for both Read and High Availability Replicas. Replica lag is measure in bytes against the current state of the primary database. For questions or concerns about the relative lag state of your replica, please reach out to Customer Support.
```
Comment (ramonguiu) [inline about/changelog.md]: @ggodeke have you confirmed this is visible yet? Maybe only part of the upcoming HA changes?

Comment (atovpeko) [inline about/changelog.md]: measure -> measured?

File: about/changelog.md [hunk 2]
```diff
@@ -8,6 +8,37 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+### Adjust chunk interval
+Customers can now adjust their chunk interval for their hypertables and continuous aggregates through the Timescale UI. In the Explorer, select the corresponding hypertable you would like to adjust the chunk interval. Under "Chunk information," you can change the chunk interval. Note that this only changes the chunk interval going forward, and does not retroactively change existing chunks.
```
Comment (atovpeko) [inline about/changelog.md]: 1. If Chunk information is a UI element name, it should be in bold, without quotes
2. In the Explorer, select the corresponding hypertable you would like to adjust the chunk interval -> Seems that this sentence is missing a preposition

File: about/changelog.md [hunk 3]
```diff
@@ -8,6 +8,37 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+### CSV import tool in the Timescale console
+We have added a CSV import tool available on service creation within the Timescale console.  For all TimescaleDB services, after service creation you can: 
+* Choose a local file
+* Select the name of the data collection to be uploaded (default is file name)
+* Choose data types for each column
+* Upload the file as a new hypertable within your service
+Look for the `Import data from .csv` tile in the `Import your data` step of service creation.
+
+### CloudWatch Permissions via Role Assumption
```
Comment (atovpeko) [inline about/changelog.md]: Seems that capitalizing each word is not necessary in this title

File: about/changelog.md (hunk 1 above)
Comment (ramonguiu) [inline about/changelog.md]:
```suggestion
Customers now have more visibility into the state of replicas running on Timescale Cloud. We’ve released a new parameter called Replica Lag within the Operations tab for both Read and High Availability Replicas. Replica lag is measure in bytes against the current state of the primary database. For questions or concerns about the relative lag state of your replica, please reach out to Customer Support.
```
Customers now have more visibility into the state of replicas running on Timescale Cloud. We’ve released a new parameter called Replica Lag within the Service Overview for both Read and High Availability Replicas. Replica Lag is measured in bytes against the current state of the primary database. For questions or concerns about the relative lag state of your replica, please reach out to Customer Support.

**Review by ramonguiu — APPROVED** [review]

**Pull request merged**